setup.cfg

```diff
@@ -2,7 +2,7 @@
 addopts =
         --strict
         --tb=native
-        weaver/
+;        weaver/
```

Comment:
shouldn't be removed, `make test` calls depend on it

Reply:
yes, sorry, I do that to run a single test from pytest. Do you have another method to target a single test? when this line is there, The whole test suite is ran.

Reply:
In pycharm, there's a green arrow next to a test, just press it and it creates a test config.
Usually, I just need to change the working dir in it to point to weaver root and all works.